src/nile/cli.py

```diff
@@ -88,26 +95,37 @@ def run(path, network):
 @click.argument("arguments", nargs=-1)
 @network_option
 @click.option("--alias")
+@watch_option
 @click.option("--abi")
 @mainnet_token_option
-def deploy(artifact, arguments, network, alias, abi=None, token=None):
+def deploy(artifact, arguments, network, alias, watch_mode, abi=None, token=None):
```

Comment:
I think we can remove the initialization in Click command methods, because the default behavior for options is passing None.